Webapp/sources/templates/_saved_pca_graphs.html

```diff
@@ -0,0 +1,16 @@
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
```

Comment:
There's already a version of this loaded in the `base.html`

Reply:
removed this from the file